src/ingestor.ts

```diff
@@ -179,14 +201,22 @@ export default async function ingestor(req) {
         type,
         payload
       }),
-      sig: body.sig
+      sig: formattedSignature
     };
     const msg = jsonParse(legacyBody.msg);
 
     if (['follow', 'unfollow', 'subscribe', 'unsubscribe', 'profile'].includes(type)) {
       legacyBody = message;
     }
 
+    if (legacyBody.address) {
+      legacyBody.address = snapshot.utils.getFormattedAddress(legacyBody.address, networkDataType);
+    }
+
+    if (legacyBody.from) {
+      legacyBody.from = snapshot.utils.getFormattedAddress(legacyBody.from, networkDataType);
+    }
```

Comment:
Don't think we need these,

Because `legacyBody.address` won't exist if it is 'follow', 'unfollow', 'subscribe', 'unsubscribe' and 'profile',  in line 196 it is using `message.from` which is already formatted because of the schema check

and for `legacyBody.from` it is from `message.from`, it is formated from schema check

Reply:
```suggestion
```